### PR TITLE
Prisma Cloud Remediation - AWS IAM Policy Misconfiguration

### PR DESCRIPTION
**Pull request by toddm92**
## Status
Ready

## Related Issues
fixes: link to the issue

## Description
Two playbooks:
- Prisma_Cloud_Remediation_-_AWS_IAM_Policy_Misconfiguration.yml
- Prisma_Cloud_Remediation_-_AWS_IAM_Password_Policy_Misconfiguration.yml

## Screenshots
Paste here any images that will help the reviewer

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------

## Required version of Demisto
(tested with) 19.10.2

## Does it break backward compatibility?
   - Yes
       - Further details:
   - No

## Must have
- [ ] Tests
- [x] Documentation (with link to it)
- [ ] Code Review

- Documentation (first attempt anyway):
https://docs.google.com/document/d/15FyVgYi8Cd8NhTfOjbUFzO2vUWNGCOAMJp9QH2YiGC8/edit?usp=sharing
- Demisto instance where playbooks live:
https://34.211.51.90/#/playbooks
- Nine (9) test incidents exist on the above Demisto instance. All tests passed.
https://34.211.51.90/#/incidents

## Dependencies
Mention the dependencies of the entity you changed as given from the precommit hooks in checkboxes, and tick after tested them.
- [ ] Dependency 1
- [ ] Dependency 2
- [ ] Dependency 3

## Additional changes
Describe additional changes done, for example adding a function to common server.

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](link)
- [ ] [CHANGELOG](link)

### Notes
Per Shai Yaakovi on merging to mater..
*"you should open the PR regularly, our content-bot will create a new branch and update the base of your PR with it"*

### YAML checklist
- [x] change the playbook version to -1
- [x] add 'fromversion: 5.0.0' line under the version field
- [x] change the id field to the name of the playbook
- [x] make sure all tasks have a description (description: "")
- [x] remove the 'sourceplaybookid' line
- [x] add the following to the end of the yml file
```
tests:
- No Test
```

### CODE REVIEW

**Comment by toddm92** [comment]
Ugh! Sorry about the extra files. I'll clean that up next PR.

**Comment by mayagoldb** [comment]
@toddm92 Please remove old files from the PR

**Comment by mayagoldb** [comment]
@yaron-libman please review the new yml files:
https://github.com/demisto/content/blob/11f06223f310ce279e2f56979eb2339f31f611ca/Playbooks/playbook-PCR_-_AWS_IAM_Password_Policy_Misconfig.yml
https://github.com/demisto/content/blob/11f06223f310ce279e2f56979eb2339f31f611ca/Playbooks/playbook-PCR_-_AWS_IAM_Policy_Misconfig.yml

File: Playbooks/playbook-PCR_-_AWS_IAM_Policy_Misconfig.yml [hunk 1]
```diff
@@ -0,0 +1,481 @@
...
+    ignoreworker: false
+  "11":
+    id: "11"
+    taskid: ee45afd1-410b-4809-87b7-abc09bfb8146
+    type: playbook
+    task:
+      id: ee45afd1-410b-4809-87b7-abc09bfb8146
+      version: -1
+      name: Prisma Cloud Remediation - AWS IAM Password Policy Misconfiguration
+      playbookName: Prisma Cloud Remediation - AWS IAM Password Policy Misconfiguration
```
Comment (mayagoldb) [inline Playbooks/playbook-PCR_-_AWS_IAM_Policy_Misconfig.yml]: Add an empty description here

File: Playbooks/playbook-PCR_-_AWS_IAM_Password_Policy_Misconfig.yml [hunk 2]
```diff
@@ -0,0 +1,592 @@
...
+        "height": 1055,
+        "width": 2280,
+        "x": -260,
+        "y": -180
+      }
+    }
+  }
+inputs:
+- key: policyId
+  value:
```
Comment (mayagoldb) [inline Playbooks/playbook-PCR_-_AWS_IAM_Password_Policy_Misconfig.yml]: input value can be empty here:
`key: policyId`
`value: {}`
`required: true`

**Pull request merged**